watcher/local/watcher_test.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2021 - See NOTICE file for copyright holders.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package local_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+
+	ethChannel "perun.network/go-perun/backend/ethereum/channel"
+	_ "perun.network/go-perun/backend/ethereum/channel/test" // For initilizing channeltest
+	"perun.network/go-perun/channel"
+	channeltest "perun.network/go-perun/channel/test"
+	"perun.network/go-perun/pkg/test"
+	"perun.network/go-perun/watcher/internal/mocks"
+	"perun.network/go-perun/watcher/local"
+)
+
+// func init() {
```

Comment:
Move to `init_test.go` and change log level to `warn` or delete it if you dont need it.

Reply:
Moved to `init_test.go`. See PR #213.